tag.go

```diff
@@ -74,9 +74,12 @@ func getLatestAcceptableTag(tagConstraint string, tags []string) (string, *Fetch
 		return latestTag, wrapError(errors.New("Tag does not exist"))
 	}
 
-	// The tag name may have started with a "v". If so, re-apply that string now
+	// go-version normalizes versions to remove the leading v and uses at least three version segments x.y.z
+	// find the origina version string that go-version matched
 	for _, originalTagName := range tags {
-		if strings.Contains(originalTagName, latestAcceptableVersion.String()) {
+		// ignore any errors since the tag was previously successfully converted
+		originalTagNameConverted, _ := version.NewVersion(originalTagName)
```

Comment:
Ignoring errors like this seems very error prone. Some future refactor of the code may break the assumption you're making in the comment. What about saving the output of the `version.NewVersion` calls above into a list and looping over that list here?